examples/pt2/README.md

```diff
@@ -0,0 +1,49 @@
+## PyTorch 2.x integration
+
+PyTorch 2.0 brings more compiler options to PyTorch, for you that should mean better perf either in the form of lower latency or lower memory consumption. Integrating PyTorch 2.0 is fairly trivial but for now the support will be experimental until the PyTorch 1.14 release.
+
+## Get started
+
+Install torchserve with nightly torch binaries
+
+```
+python ts_scripts/install_dependencies.py --cuda=cu117 --nightly_torch
+pip install torchserve torch-model-archiver
+```
+
+## Package your model
+
+PyTorch 2.0 supports several compiler backends and you pick which one you want by passing in an optional file `compile.json` during your model packaging
+
+`{"pt2" : "inductor"}`
+
+As an example let's expand our getting started gui  de with the only difference being passing in the extra `compile.json` file
+
+```
+mkdir model_store
+torch-model-archiver --model-name densenet161 --version 1.0 --model-file ./serve/examples/image_classifier/densenet_161/model.py --export-path model_store --extra-files ./serve/examples/image_classifier/index_to_name.json,./serve/examples/image_classifier/compile.json --handler image_classifier
+torchserve --start --ncs --model-store model_store --models densenet161.mar
+```
+
+The exact same approach works with any other model, what's going on is the below
```

Comment:
@msaroufim Training part may need a short explanation to make it clear it is not related to inference and I am guessing without the optimized training we can still do the optimized inference. Maybe we can just keep it for inference here and blow that you are explaining mention the Training optimizations as well.